sdk/appconfiguration/azure-appconfiguration-provider/dev_requirements.txt

```diff
@@ -1,5 +1,5 @@
 -e ../../core/azure-core
-../../appconfiguration/azure-appconfiguration
+azure-appconfiguration==1.6.0b2
```

Comment:
Is it possible to avoid beta version dependency?

Reply:
Moved this back to using the local one.